firebase/build.gradle

```diff
@@ -32,12 +32,12 @@ dependencies {
     testImplementation "com.rudderstack.android.sdk:core:[1.7.0, 1.12.0]"
 
     // RudderStack Android SDK
-    compileOnly 'com.rudderstack.android.sdk:core:[1.7.0, 1.12.0]'
+    compileOnly 'com.rudderstack.android.sdk:core:[1.15.0, 2.0.0)'
```

Comment:
Can we use the current latest version as the lower bound here ?

Reply:
Implemented.